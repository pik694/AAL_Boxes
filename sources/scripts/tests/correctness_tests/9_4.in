9
84 64 93
99 34 50
33 61 34
69 62 83
38 14 1
24 66 70
46 8 2
57 99 26
45 62 92

