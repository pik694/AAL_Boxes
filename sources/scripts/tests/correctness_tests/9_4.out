4 2 1 6 5 3 0 7 8 
1607330->1071666
